components/wpt-results.html

```diff
@@ -146,40 +153,31 @@
         testFileResults.forEach(result => {
           const testFiles = result.testFiles
           const resultsURL = result.resultsURL
+          const encodedResultsURL = btoa(result.resultsURL)
```

Comment:
What is result.resultsURL? Is it the URL for the JSON that contains results for one run? And what's the advantage of using encodedResultsURL over resultsURL below?